### PR TITLE
Dark color scheme for Lightly

### PR DESCRIPTION
**Pull request by vikdevelop**
Hi, Luwx,
I love the style of Qt Lightly apps because it's minimalistic, modern, and much nicer than Breeze by KDE. But unfortunately it doesn't have a dark color scheme, so I decided to modify Lightly's color scheme to dark, below is the result. I would be very happy if you integrated this color scheme into this repository. What do you think?

### CODE REVIEW

**Comment by jonhteper** [comment]
Hi, i tried this color scheme and I find some inconsistencies: 
*this type list,
bug-lightly
* the top bar color in unfocus windows,
bug-lightly-topbar
* and the tooltips
bug-lightly-tooltips

**Comment by Luwx** [comment]
Thanks for the PR.

I recommend making the color scheme from an existing dark one, otherwise you'll probably still have some bits of the light color scheme, as @jonhteper mentioned.

In general, I feel the colors are too dark, you could make the main background color brighter like #222222 and the view color a little above that. Having the view color darker than the background results in muddy looks when the shadows are enabled.

File: colors/Lightly-Dark.color [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+ForegroundInactive=112,112,112
+ForegroundLink=82,118,217
+ForegroundNegative=223,49,49
+ForegroundNeutral=236,145,39
+ForegroundNormal=223,223,223
+ForegroundPositive=34,161,87
+ForegroundVisited=139,74,176
+
+[General]
+Name=lightly dark
```
Comment (Luwx) [inline colors/Lightly-Dark.color:92]: Name should be capitalized.

Comment (vikdevelop) [reply colors/Lightly-Dark.color]: I Updated Lightly Dark Color scheme.

**Comment by arun54321** [comment]
It is not working. file extension should be .colors not .color

**Comment by logant547** [comment]
Hi there, just to let you all know I've made a full colour scheme to follow the original style. I've also opened a pull request.

**Pull request closed without merging**